tests/javaRosa.js

```diff
@@ -595,6 +597,31 @@ require([
         });
     });
 
+    describe("The javaRosaplugin with one language", function() {
+        before(function(done) {
+            util.init({
+                javaRosa: { langs: ['en'] },
+                core: {
+                    onReady: function () {
+                        done();
+                    }
+                }
+            });
+        });
+
+        it("should replace the default form with placeholder when cleared", function(){
+            util.loadXML("");
+            util.addQuestion('Trigger', 'label');
+            util.clickQuestion('label');
+            $('[name=itext-en-label]').val('blah').change();
+            $('.itext-block-label-add-form-image').click();
+            $('[name=itext-en-label]').val('').change();
+            util.assertXmlEqual(call("createXML"), 
+                                NO_LABEL_TEXT_ONE_LANG_XML,
+                                {normalize_xmlns: true});
```

Comment:
When I manually do the steps performed by this test I see a placeholder in the label textarea, but Tools > Edit Source XML shows an empty `<value></value>`. That seems like a bug.